common/retinue_subunits/00_retinue_subunits.txt

```diff
@@ -1260,6 +1269,12 @@ wc_ret_cul_twilhammer = {
 	potential = {
 		# is_tribal = no
 		culture = twilights_hammer
+		
+		FROM = {
+			ROOT = {
+				prev_is_visible_necromancer_to_this_trigger = yes
```

Comment:
I can transfer feudal necromancer vassal with this retinue to the Archbishop of the Light
Then, after death of said necro, Archbishop will use armies of undeads in holy wars
LOOKS NICE